packages/commonwell-sdk/src/models/address.ts

```diff
@@ -40,3 +43,65 @@ export function normalizeStatePreprocess(arg: unknown): unknown {
   if (typeof arg === "string") return normalizeState(arg);
   return undefined;
 }
+
+export function normalizeStatePreprocessSafe(arg: unknown): unknown {
+  if (typeof arg === "string" && ["", "undefined", "null"].includes(arg.trim())) return undefined;
```

Comment:
we could use `emptyStringToUndefinedSchema` so we don't need to add the check for null and undefined strings here